rotkehlchen/tests/unit/globaldb/test_globaldb.py

```diff
@@ -3,6 +3,7 @@
 from pathlib import Path
 from shutil import copyfile
 from typing import TYPE_CHECKING
+from unittest.mock import patch
```

Comment:
why is this needed here now?

Reply:
Is me working in 3 things and having the wrong import commited. Has had to be after running the linter.sry